operator/gitops/argocd/pipeline-service/openshift-pipelines/osp-nightly-catalog-source.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     argocd.argoproj.io/sync-wave: "0"
 spec:
   sourceType: grpc
-  image: quay.io/openshift-pipeline/openshift-pipelines-pipelines-operator-bundle-container-index@sha256:c2c6587e059b0b5144f4b2cff79f31f1f6fee36f0927b301a17a3b608237134f
+  image: quay.io/openshift-pipeline/openshift-pipelines-pipelines-operator-bundle-container-index@sha256:beec206d5f9650173348c0d9db404faee6b791ec6d25a9ea3410a909a8e37187
```

Comment:
no objections @enarha to merge

that said, can you put the gitlab URL corresponding to this sha that tells us what versions of everything is getting pulled in

Reply:
I'll do tomorrow. But from a quick look, it's a build 8 days old.

Reply:
@enarha looked on the console in stage and confirmed the operator is listing the correct version:
```
Components
Tekton Pipelines: v0.59.0
Tekton Triggers: v0.26.2
Pipelines as Code: v0.27.0
Tekton Chains: v0.20.1
Tekton Hub (tech-preview): v1.16.0
Tekton Results (tech-preview): v0.9.2
```